src/native/libs/System.Security.Cryptography.Native.Apple/entrypoints.c

```diff
@@ -30,6 +30,7 @@ static const Entry s_cryptoAppleNative[] =
     DllImportEntry(AppleCryptoNative_AesGcmDecrypt)
     DllImportEntry(AppleCryptoNative_ChaCha20Poly1305Encrypt)
     DllImportEntry(AppleCryptoNative_ChaCha20Poly1305Decrypt)
+    DllImportEntry(AppleCryptoNative_IsAuthenticationFailure)
```

Comment:
These are loosely alpha sorted. We should put it under `AppleCryptoNative_HmacOneShot`